playground/AzureContainerApps/AzureContainerApps.AppHost/AzureContainerApps.AppHost.csproj

```diff
@@ -6,6 +6,8 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <IsAspireHost>true</IsAspireHost>
+    <!-- This is an apphost project, doesn't need a Newtonsoft.Json reference -->
+    <AzureCosmosDisableNewtonsoftJsonCheck>true</AzureCosmosDisableNewtonsoftJsonCheck>
```

Comment:
The new Azure Cosmos package added a target that runs on restore and surfaces an error when there is no direct dependency on Newtonsoft.Json. Full details can be found here: https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4839 but TL;DR is that they internally depend on newtonsoft but don't want to flow that transitive dependency and instead allow end-user applications to control the version they need.

In our repo, we disable this warning on all places where Newtonsoft isn't needed (as the Cosmos SDK won't run), and we reference Newtonsoft directly when necessary.

Reply:
Why is the AppHost project depending on the Cosmos SDK? Is it for the health check? If so, then I think we actually DO want Newtonsoft to be present, or else the AppHost is going to fail the health check when Newtonsoft tries to get loaded and it isn't in the bin directly.